src/services/language.ts

```diff
@@ -140,7 +140,8 @@ export class LanguageService {
 
   convertDate(date){
     if(this._lang === 'fa'){
-      momentJalali.loadPersian(true);
+      // momentJalali.loadPersian(true);
+      momentJalali.loadPersian();
```

Comment:
Sorry for late. Resolve this issue. I remove 'jalali-moment' package and install it again. This function need one Boolean parameter.